Changed CredentialsType to CredentialType.

dotps1 opened the pull request:
Updated the Set-DscResource to use CredetinalType rather then CredentialsType.  This corrects the `the data is invalid.` error I get when creating Subscriptions.  I have updated the README.md to reflect this change.  I also escaped the Query example in the README.md to prevent the markdown from being rendered as italicized text.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/powershell/xwindowseventforwarding/10)
<!-- Reviewable:end -->

dotps1 committed:
Corrected CredentialType ParameterName and XML Tag

When generating the xml document, the proper tag name is `<CredentialType>` not `<CredetnialsType>`.  the 's' has been removed.  This should fix `The data is invalid.` error that is thrown when generating the subscription.

dotps1 committed:
Corrected variable type in CredentialType XML Tag

dotps1 committed:
Updated README.md to reflect change in CredentialType tag

Updated the README.md documentation to reflect typo fix for CredentialsType to CredentialType.  Also put back ticks around the query example so markdown formatting is not being rendered and italicizing the text.

msftclas commented:
Hi __@dotps1__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, MSBOT;

dotps1 committed:
Fix CredentialType typo

dotps1 committed:
Update CredentialsType to CredentialType in Get/Test functions

Commit:
Default TransportPort to null

Errors throw when defaulting the TransportPort to 5985, so defaulting to null.  Also, the TransportPort was not defined in the Get-TargetResource, so I added that.  I also fixed some of the appveyor failed tests.

Commit:
Added new line to NewModuleScript, removed tabs from readme.

dotps1 commented:
ok, after more research, the tag was correctly named, however, it should only be used on a CollectorInitiated subscription.  I will submit a new corrected pull request.
